catppuccin-theme.el

```diff
@@ -582,6 +582,31 @@ If called non-interactively, the FLAVOR must be one of 'frappe, 'latte, 'macchia
                                            (list :height catppuccin-height-title-3)))
                (elpher-gemini-preformatted :inherit fixed-pitch
                                            :foreground ,ctp-green)
+               ;; erc
+               (erc-action-face :foreground ,ctp-green)
+               (erc-bold-face :weigth bold)
+               (erc-button :inherit link)
+               (erc-command-indicator-face :foreground ,ctp-mauve)
+               (erc-current-nick-face :foreground ,ctp-peach)
+               (erc-dangerous-host-face :background ,ctp-red :foreground ,ctp-text)
+               (erc-default-face :foreground ,ctp-text)
+               (erc-direct-msg-face :foreground ,ctp-rosewater)
+               (erc-error-face :foreground ,ctp-red :weight bold)
+               (erc-fool-face :foreground ,ctp-subtext0)
+               (erc-header-line :background ,ctp-crust :foreground ,ctp-text)
+               (erc-input-face :foreground ,ctp-rosewater)
+               (erc-inverse-face :background ,ctp-text :foreground ,ctp-base)
+               (erc-italic-face :slant italic)
+               (erc-keyword-face :foreground ,ctp-lavender :weight bold)
+               (erc-my-nick-face :foreground ,ctp-lavender :weight bold)
+               (erc-my-nick-prefix-face :foreground ,ctp-lavender :weight bold)
+               (erc-nick-default-face :foreground ,ctp-blue :weight bold)
+               (erc-nick-prefix-face :foreground ,ctp-blue :weight bold)
+               (erc-nick-msg-face :foreground ,ctp-yellow :weight bold)
+               (erc-notice-face :foreground ,ctp-subtext0 :slant italic)
+               (erc-pal-face :foreground ,ctp-pink :weight bold)
+               (erc-prompt-face :foreground ,ctp-blue :weight bold)
+               (erc-timestamp-face :foreground ,ctp-teal)
```

Comment:
These faces are used for colour code highlighting, these were the colours I used
```suggestion
               (erc-timestamp-face :foreground ,ctp-teal)
               (bg:erc-color-face0 :background ,ctp-text)
               (bg:erc-color-face1 :background ,ctp-crust)
               (bg:erc-color-face2 :background ,ctp-blue)
               (bg:erc-color-face3 :background ,ctp-green)
               (bg:erc-color-face4 :background ,ctp-red)
               (bg:erc-color-face5 :background ,ctp-maroon)
               (bg:erc-color-face6 :background ,ctp-mauve)
               (bg:erc-color-face7 :background ,ctp-peach)
               (bg:erc-color-face8 :background ,ctp-yellow)
               (bg:erc-color-face9 :background ,ctp-green)
               (bg:erc-color-face10 :background ,ctp-lavender)
               (bg:erc-color-face11 :background ,ctp-teal)
               (bg:erc-color-face12 :background ,ctp-sapphire)
               (bg:erc-color-face13 :background ,ctp-pink)
               (bg:erc-color-face14 :background ,ctp-overlay2)
               (bg:erc-color-face15 :background ,ctp-text)
               (fg:erc-color-face0 :foreground ,ctp-text)
               (fg:erc-color-face1 :foreground ,ctp-crust)
               (fg:erc-color-face2 :foreground ,ctp-blue)
               (fg:erc-color-face3 :foreground ,ctp-green)
               (fg:erc-color-face4 :foreground ,ctp-red)
               (fg:erc-color-face5 :foreground ,ctp-maroon)
               (fg:erc-color-face6 :foreground ,ctp-mauve)
               (fg:erc-color-face7 :foreground ,ctp-peach)
               (fg:erc-color-face8 :foreground ,ctp-yellow)
               (fg:erc-color-face9 :foreground ,ctp-green)
               (fg:erc-color-face10 :foreground ,ctp-lavender)
               (fg:erc-color-face11 :foreground ,ctp-teal)
               (fg:erc-color-face12 :foreground ,ctp-sapphire)
               (fg:erc-color-face13 :foreground ,ctp-pink)
               (fg:erc-color-face14 :foreground ,ctp-overlay2)
               (fg:erc-color-face15 :foreground ,ctp-text)
```
Now if you look in `M-x customize-group erc-faces` you'll see that all erc faces are themed.